bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -470,22 +489,26 @@ class VideoProvider extends Component {
 
     const isCurrent = id === this.props.userId;
     const peer = this.webRtcPeers[id];
-    const { peerConnection } = peer;
 
     const attachVideoStream = () => {
```

Comment:
It might be a good idea to rename this to `attachVideoStreamHelper` or something like it so we don't confuse it with the outer function